client/my-sites/posts/post-controls.jsx

```diff
@@ -190,6 +198,10 @@ export const PostControls = props => {
 
 	return (
 		<div className={ classes }>
+			<QuerySharePostActions
```

Comment:
To fix the "To fix" in the PR description, shouldn't it be sufficient to add `{ this.props. isPublicizeEnabled && <Query... }`?

Reply:
I guess you are right :)